database/seeders/DatabaseSeeder.php

```diff
@@ -13,8 +13,8 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        $this->call([
-            SheetTableSeeder::class,
-        ]);
+        // $this->call([
+        //     SheetTableSeeder::class,
+        // ]);
```

Comment:
Station13 あたりでこの全文コメントアウトを解除すれば OK
……というのは「実践的教材」としては若干イケてないな〜と思うので、
調べたらなに書けばいいかわかる、の ↓ ぐらいでどうでしょう？
```suggestion
        $this->call([
            // ここに Seeder を追加する
        ]);
```

Reply:
suggestionありがとうございます！
ありがたく頂戴します！